src/Commands/Site/ImportCommand.php

```diff
@@ -0,0 +1,40 @@
+<?php
+namespace Pantheon\Terminus\Commands\Site;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Pantheon\Terminus\Site\SiteAwareInterface;
+use Pantheon\Terminus\Site\SiteAwareTrait;
+use Terminus\Exceptions\TerminusException;
+
+class ImportCommand extends TerminusCommand implements SiteAwareInterface
+{
+    use SiteAwareTrait;
+    /**
+     * Imports a site archive onto a Pantheon site
+     *
+     * @authorized
+     *
+     * @name site:import
+     * @aliases import
+     *
+     * @option string $site Name of the site to import to
+     * @option string $url  URL at which the import archive exists
+     * @usage terminus import <site_name> <archive_url>
+     *   Imports the file at the archive URL to the site named.
+     */
+    public function import($sitename, $url)
+    {
+        $site = $sitename.'.dev';
```

Comment:
Instead of appending this '.dev', you should:
`$this->getSiteEnv($sitename, 'dev')`